testapp/models.py

```diff
@@ -6,3 +6,10 @@
 class File(models.Model):
     file = PrivateMediaFileField()
     image = PrivateMediaImageField()
+
+
+class File2(File):
+    """Used for testing `admin.PrivateMediaMixin.no_download_fields`"""
+
+    class Meta:
+        proxy = True
```

Comment:
I suspect this may complain about missing migrations, but I'm not sure.

Use `DJANGO_SETTINGS_MODULE=testapp.settings django-admin makemigrations` to check